README.md

```diff
@@ -48,6 +48,11 @@ provides the following:
   are self-contained from the [Python Build Standalone](
   https://python-build-standalone.readthedocs.io/en/latest/) project.
 
++ Support for `.env` files:
+
+  The first `.env` file found in the current directory or any of its parent directories is loaded
```

Comment:
How far up the chain does this go?

Reply:
All the way until it finds one or none. Just like our `pants.toml` search: https://crates.io/crates/dotenv

So repo `.env` overrides user home `.env` (in typical arrangements), etc.

Reply:
Oh, wild, I didn't know that Pants did that today - those those searches stopped at project root